packages/framework/fluid-static/src/serviceAudience.ts

```diff
@@ -112,12 +112,20 @@ export abstract class ServiceAudience<M extends IMember = IMember>
   /**
    * {@inheritDoc IServiceAudience.getMyself}
    */
-  public getMyself(): M | undefined {
+  public getMyself(): IMyself<M> | undefined {
     const clientId = this.container.clientId;
     if (clientId === undefined) {
       return undefined;
     }
-    return this.getMember(clientId);
+
+    const member = this.getMember(clientId);
+    if (member === undefined) {
+        return undefined;
```

Comment:
I don't think we need to check the member undefined condition. look at the `getMember()` implementation.

Reply:
I believe this could cause some typing problems since IMyself doesn't expect undefined fields. 
![image](https://user-images.githubusercontent.com/112977307/196230641-b4c47e54-bbcc-4ae1-9734-518a84fde186.png)